tools/benchmarksAnalyzer/src/main/kotlin/org/jetbrains/analyzer/SummaryBenchmarksReport.kt

```diff
@@ -133,6 +133,34 @@ class SummaryBenchmarksReport (val currentReport: BenchmarksReport,
         }
     }
 
+    enum class Metric(val suffix: Regex) {
+        EXECUTION_TIME(Regex("(?<!(\\.(codeSize|compileTime)))$")),
```

Comment:
Why codeSize?

Reply:
Execution time has no suffix. So just negative lookbehind assertion to check that there is no suffix.

Reply:
As we control both sides, maybe makes sense to introduce more reliable reporting mechanism here?

Reply:
Do you suggest  to create one extra property of Benchmark - metric type?